docs/manual/external-tools.tex

```diff
@@ -767,20 +767,25 @@ \section{IntelliJ IDEA\label{intellij}}
   \<\$CHECKERFRAMEWORK/checker/dist/javac.jar>.
 
 \item
-  Add the annotated JDK library.
+  Add the annotated JDK library using the following method.
   Go to the ``Settings'' menu via (File>Settings) or (Ctrl-Alt-S) and
   navigate to the ``Java Compiler'' sub-menu via (Build, Execution,
   Deployment>Compiler>Java Compiler).  Add the following to the field
   ``Additional command line parameters'': \\
   \<-Xbootclasspath/p:\$CHECKERFRAMEWORK/checker/dist/jdk8.jar>
+  Note: It is a good idea to enclose the address in quotation
+  marks to escape special characters and spaces.
+  E.g. \<-Xbootclasspath/p:"\$CHECKERFRAMEWORK/checker/dist/jdk8.jar">
```

Comment:
How about making this change on line 775, so you don't have to add lines 776-778?

Reply:
Sure thing. I'd like to leave the note on the special characters in however, I discussed with @wmdietl  and I think it would be valuable information for an inexperienced developer trying to use the tool.

Reply:
I don't think that the Checker Framework manual is a place to include general advice for inexperienced programmers.  Let's just give the exact text that the programmer needs to type in -- and let's not give the incorrect information first with the correct information in a note afterward.  

Also, the formatting of the note on the same line with `.../jdk8.jar` is confusing; you would have wanted a line break, but just removing lines 776-778 resolves the problem.

Reply:
Sure thing. I'll make those changes. Thank you